src/pytest_memray/marks.py

```diff
@@ -1,34 +1,65 @@
 from __future__ import annotations
 
+from dataclasses import dataclass
+from typing import Optional
+from typing import Tuple
+
 from memray import AllocationRecord
 
 from .utils import parse_memory_string
 from .utils import sizeof_fmt
 
+PytestSection = Tuple[str, str]
+
+
+# REVIEW bikeshed the name?
+@dataclass
+class FailedTestMemoryInfo:
```

Comment:
Let's remove the comment, I think we can keep it just `_MemoryInfo` (can be private, outside people don't need to use it) 👍